extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -136,6 +136,7 @@ private synchronized void initializeProperties() {
                 }
                 updateProperty(LIGHT_UNIQUE_ID, fullLight.getUniqueID());
                 isOsramPar16 = OSRAM_PAR16_50_TW_MODEL_ID.equals(modelId);
+                updateThing(thing);
```

Comment:
How is this change related to the `AutomaticInboxProcessor registered as an RegistryListener`?

Reply:
see above